twake/frontend/src/app/services/RouterServices.tsx

```diff
@@ -23,19 +23,20 @@ class RouterServices {
   history = createBrowserHistory();
 
   // Define your route here
-  pathnames: Readonly<Pathnames> = {
+  public pathnames: Readonly<Pathnames> = {
     CLIENT: '/client',
+    CLIENT_APP: '/client/w/:workspace/c/:channel',
```

Comment:
do we really need `/client` prefix?

Reply:
"client" yes but "w" not @stephanevieira75 :) 
We stole it from Slack, it'll be needed in the future to have a constant prefix (we wanted a smaller prefix, but web.twake.app/app or web.twake.app/web seems a bit redundant !)

Reply:
Removed :+1: